src/Extension/Evaluator.cs

```diff
@@ -23,5 +27,35 @@ public Evaluation EvaluateResult(KernelCommandResult result)
             }
             return new Evaluation { Passed = true };
         }
+
+        public void AddInputExecutionCriterion(int cellId, string code)
+        {
+            executionCriteria.Remove(cellId);
```

Comment:
"execution" is comput-speak. 

What's an appropriate term for his in our domain?

Reply:
Implicit removal on add seems surprising. Would we not want to allow more than one criterion?

Reply:
Modified to allow many criterions to be attached to a question